src/features/targetedOutreach/components/OutreachPopup/index.tsx

```diff
@@ -61,6 +61,7 @@ const OutreachPopup = (): ReactElement | null => {
       })
     }
     dispatch(closeOutreachBanner())
+    refetch()
```

Comment:
I also looked into setting `keepUnusedDataFor: 0` or `refetchOnMountOrArgChange: 0` but then the banner flickers when reconnecting the wallet since the query is in the process of updating after invalidating the cache.

Reply:
Yes, I changed the `createSubmission` call into a mutation and invalidated the getSubmission query whenever it is called.